config/symbols.us.dra.txt

```diff
@@ -11,4 +11,5 @@ g_PlOvlAluBatSpritesheet = 0x8013C004;
 g_PlOvlSpritesheet = 0x8013C020;
 g_imgUnk8013C200 = 0x8013C200;
 g_imgUnk8013C270 = 0x8013C270;
+g_ServantDesc = 0x80170000;
```

Comment:
I think the idea is to eventually be able to build without splat, but this makes it so we can't get this symbol at this address without it. Originally there was either a custom linker script or it's just hard coded casts of 0x80170000. @Xeeynamo any opinion here?

Reply:
`ld` supports the `--just-symbols` option for providing a symbols file like is currently being used. I'm not sure if it's more or less flexible to continue to have have platform specific symbol tables outside of the code. That's not dissimilar from the current linker command and doesn't affect building with or without splat, AFAICT.

https://github.com/Xeeynamo/sotn-decomp/blob/5fa4630311eed05e31fb5075da490135a39a6b6e/Makefile#L116-L123